.gitignore

```diff
@@ -30,7 +30,7 @@ coverage
 
 # Grunt intermediate storage (https://gruntjs.com/creating-plugins#storing-task-files)
 .grunt
-
+git 
```

Comment:
Why need gitignore `git` ah? The original `.git` should be automatically ignored.

Reply:
I think i typed this by accident as I was trying to commit n push